node/impl/storminer.go

```diff
@@ -725,11 +725,13 @@ func (sm *StorageMinerAPI) DagstoreInitializeAll(ctx context.Context, params api
 
 	go func() {
 		for i, k := range toInitialize {
-			select {
-			case <-throttle:
-				// acquired a throttle token, proceed.
-			case <-ctx.Done():
-				return
+			if throttle != nil {
+				select {
+				case <-throttle:
```

Comment:
There's a send to the `throttle` channel that will block below if the channel is `nil`.

Reply:
https://github.com/filecoin-project/lotus/pull/7177.